test/index.ts

```diff
@@ -0,0 +1,93 @@
+const hre = require("hardhat");
+const { expect } = require("chai");
+const { MerkleTree } = require("merkletreejs");
+const keccak256 = require("keccak256");
+
+const sampleData: { string: string } = require("./sample-data.json");
+
+async function deploy(name: any, ...params: any) {
+  const Contract = await hre.ethers.getContractFactory(name);
+  return await Contract.deploy(...params).then((f: any) => f.deployed());
+}
+
+function hashToken(account: string, value: string) {
+  return Buffer.from(
+    hre.ethers.utils
+      .solidityKeccak256(["address", "uint256"], [account, value])
+      .slice(2),
+    "hex"
+  );
+}
+
+describe("Create ERC20 Merkle Drop and test", function () {
+  let factory: any;
+  before(async function () {
+    factory = await deploy("MerkleFactory");
+    this.accounts = await hre.ethers.getSigners();
+    this.merkleTree = new MerkleTree(
```

Comment:
This makes me feel like anyone (non-snapshot users) can create a clone and use our factory, not sure this is correct approach

Reply:
https://github.com/snapshot-labs/drop/pull/1#discussion_r777112181